src/lib/components/ui/Search.svelte

```
@@ -0,0 +1,199 @@
+			};
+		};
+		_rankingInfo?: RankingInfo | undefined;
+		_distinctSeqID?: number | undefined;
+	};
+
+	const results = writable<SearchResult[]>([]);
+	const loading = writable<boolean>(false);
+
+	let query = '';
+
+	let debounceTimeout: NodeJS.Timeout;
```

ESLint

[error] 'NodeJS' is not defined.

(no-undef)